### PR TITLE
Object sanity

### PR DESCRIPTION
**Pull request by JamesCoburn246**

### CODE REVIEW

**Comment by JamesCoburn246** [comment]
I am curious to see what people think about the numbers on this one.

File: components/unstable_strafing_ground.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 const DIRECTION = {"LEFT":-1, "RIGHT": 1};
+const MULTIPLIERS = {
+    LOW: 0.5,
+    HIGH: 1.5,
+}
 
 Crafty.c("UnstableMovementGround", {
```
Comment (StevenR152) [inline components/unstable_strafing_ground.js:7]: Rename the component too?

**Review by StevenR152 — APPROVED** [review]

File: components/spike_bush.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                    this.addComponent("bush_spiky");
+                    this.removeComponent("bush_berries");
+                    this.color('red');
+                    //this.y += 124;
+                    this.lethal = true;
+                }
+            }
+        });
+
+        this.onHit("Spike", function () {
```
Comment (TomasMorton) [inline components/spike_bush.js:28]: I was wondering when I woke up whether we could do the collision checks on the bush instead of the player, thanks!

Comment (JamesCoburn246) [reply components/spike_bush.js:28]: Awesome

**Pull request merged**